docs/tutorial/kymotracking.rst

```diff
@@ -58,12 +58,12 @@ Considering that we only want to run the kymotracker on part of the image, we al
 In this case, we track particles between 0 and 350 seconds, and 7 and 22 micron.
 Running the algorithm is easy using the function :func:`~lumicks.pylake.track_greedy`::
 
-    traces = lk.track_greedy(kymo, "green", line_width=3, pixel_threshold=3, window=6, rect=[[0, 7], [350, 22]])
+    traces = lk.track_greedy(kymo, "green", line_width=0.3, pixel_threshold=3, window=6, rect=[[0, 7], [350, 22]])
 
 The result of tracking is a list of kymograph traces::
 
     >>> print(len(traces))  # the number of traces found in the kymo
-    7411
+    7417
```

Comment:
Note that this was already `7417` on the master branch, it was incorrect in the docs.